saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/cookiePolicy.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Cookie policy message
+ * 
+ * @author Łukasz Pawełczak
+ */
+
+var CookiePolicy = (function() {
+    
+    var scope = {},
+        defaultOptions = {
+            buttonShowMessageId: "#cookie-show-message",
+            buttonAcceptId: "#cookie-accept",
+            messageId: "#cookie-message",
+            windowId: "#cookie-window"
+        },
+        
+        options = {},
+        
+        cookieName = "cookiePolicyAccepted",
+        cookieTime = 7 * 24 * 60; // 7 days
+    
+    
+    scope.init = function(arguments) {
+        
+        options = arguments || defaultOptions;
+        
+        initButtonShowCookiePolicy();
+        initButtonAcceptCookiePolicy();
+        
+    };
+    
+    return scope;
+    
+    
+    /*
+     * Show cookie policy message
```

Comment:
While describing methods the 3rd person form of a verb should be used. Here: Shows